### ISSUE TITLE
25 January homepage content programming

### ISSUE THREAD

**Issue by devaneymoz** [mozilla/addons-frontend#4163]
For the **Thursday, 25 January** push, please update the AMO homepage programming in accord with what's outlined [here](https://docs.google.com/document/d/1g4RuGKW__eTHNd277rKPMM46ygonXzW68jSs2OXEkPM/edit#heading=h.e9p8yhnuv39m).

@EnTeQuAk @muffinresearch @diox

**Comment by muffinresearch** [comment]
@tofumatt let me know if you need a hand to get this done, we could potentially split it up.

**Comment by tofumatt** [comment]
Sorry, forgot to reply, but shouldn't be an issue. Just getting to it now but it should be ready soon. 😄

**Comment by ioanarusiczki** [comment]
@tofumatt I verified the new extensions added in hero/carousel on AMO dev FF57 (Win 10 and Android 8.0)
I've noticed that the banners display a shorter name for two extensions.
"Firefox Multi Account" containers is displayed as "Multi Account container" https://addons.mozilla.org/en-US/firefox/addon/multi-account-containers/ and "Grammarly for Firefox" is displayed as "Grammarly" https://addons.mozilla.org/en-US/firefox/addon/grammarly-1/ 
I don't know if this is the expected for the two featured extensions mentioned above.

![banners](https://user-images.githubusercontent.com/33448286/35270975-4d14229a-0039-11e8-95de-1bb73b22cb22.png)



Also, when some of these extensions are clicked on AMO dev the page is not displayed considering that these are featured extension on AMO prod. I will re-check this issue after the push.
